### PR TITLE
Update images and small UI changes

### PR DESCRIPTION
**Pull request by Pive01**
## Content
- Removed card and left box as the surrounding of components so it's more blended with the background like other extensions do
- Added an update button that runs `localstack update docker-images`
- Made header and tab panels sticky so when you scroll (i.e. on logs) you can interact with them without scrolling all way up
- Switched to version 1.3.0 for health (localhost/_localstack/health  instead of localhost/health)
## Preview
Screenshot from 2022-12-09 10-10-18
Screenshot from 2022-12-09 10-10-24
Screenshot from 2022-12-09 10-10-46

### CODE REVIEW

File: ui/src/components/Header/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  };
+
+  const handleUpdateClick = () => {
+    handleClose();
+    setOpenModal(true);
+  };
+
+  return (
+    <div>
+      {openModal && <UpdateDialog open={openModal} onClose={() => setOpenModal(false)} />}
```
Comment (lukqw) [inline ui/src/components/Header/Menu.tsx:29]: seems like the conditional rendering is superfluous given the `open={openModal}` ...

**Comment by lukqw** [comment]
more general feedback:
image

maybe we should hide either the `start` or `stop` button, depending on if LS is running?
or are we supporting starting multiple instances?

**Comment by Pive01** [comment]
> more general feedback: image
> 
> maybe we should hide either the `start` or `stop` button, depending on if LS is running? or do we support starting multiple instances?

This was one of the points in the email from the docker team...preparing another PR to apply all suggested changes (this included) and maybe merge start and stop button in one and change it depending on the state?

**Comment by lukqw** [comment]
Sounds good to me! 👍

**Pull request merged**